x-pack/test_serverless/api_integration/test_suites/observability/common_configs/config.group1.ts

```diff
@@ -24,6 +24,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       require.resolve('../../common/scripts_tests'),
       require.resolve('../../common/search_oss'),
       require.resolve('../../common/search_xpack'),
+      require.resolve('../../common/core'),
```

Comment:
The main `index` file was removed in favor of using common configs ([#167172](https://github.com/elastic/kibana/pull/167172)) to specify the common tests to run on each project.